assets/stylesheets/govuk-elements/_buttons.scss

```diff
@@ -45,3 +45,8 @@
     background-image: file-url("icons/icon-pointer-2x.png");
   }
 }
+
+.button + .inline-options {
+    padding-top: em(8);
```

Comment:
Added a commit to set the line-height to the same as the button, and the padding to match.

Looks better now.
![screenshot from 2015-08-11 12 42 10](https://cloud.githubusercontent.com/assets/117398/9196764/5d6c6fec-4026-11e5-8288-60cb8d898ea8.png)